portfolio/src/main/webapp/index.html

```diff
@@ -30,68 +30,83 @@
     <script src="task.js"></script>
     <script src="calendar.js"></script>
     <script src="request.js"></script>
-    <script src="calendarAPI.js"></script>
-    <script src="sheetsApi.js" onload="handleClientLoad()"></script>
     <script src="https://apis.google.com/js/api.js"></script>
+    <script src="calendarApi.js"></script>
+    <script src="sheetsApi.js"></script>
   </head>
-  <body>
+ 
+  <body onload="useTodayAsDefault(); setClosestEventTime()">
     <div class="container-fluid">
       <nav class="navbar navbar-expand-lg navbar-light bg-light">
         <a class="navbar-brand" href="#">Task Scheduler</a>
-        <div class="collapse navbar-collapse" id="navbar-supported-content">
-          <ul class="navbar-nav mr-auto">
+        <div class="collapse navbar-collapse" id="navbarSupportedContent">
```

Comment:
Id's should be hyphen separated.

Source: https://google.github.io/styleguide/htmlcssguide.html#ID_and_Class_Name_Delimiters

Reply:
Updated. I actually deleted the ID because this ID is not necessary.